linode/manager
# refactor: [M3-7531] - SideMenu & UserMenu Storybook v7 Story

## Description 📝
Small PR to refactor the `SideMenu` & `UserMenu` `.mdx` into v7 `.tsx` stories. It does not affect client code, UI or the user experience.

## Changes  🔄
- Move SideMenu inside the PrimaryNav directory and update imports
- Refactor the `SideMenu` `.mdx` into a v7 `.tsx` story 
- Refactor the `UserMenu` `.mdx` into a v7 `.tsx` story 

## Preview 📷
![Screenshot 2023-12-04 at 16 15 21](https://github.com/linode/manager/assets/130582365/bc1d11b6-a60a-450c-be97-1a19bbce2169)

![Screenshot 2023-12-04 at 16 15 10](https://github.com/linode/manager/assets/130582365/80c13a84-d4f9-4097-9c88-e17ec57c8e16)

## How to test 🧪

### Verification steps 
- pull code locally and run `yarn storybook`
- verify http://localhost:6006/?path=/docs/features-navigation-side-menu--documentation
- verify http://localhost:6006/?path=/docs/features-navigation-user-menu--documentation

## As an Author I have considered 🤔

*Check all that apply*
